src/WebJobs.Extensions.ApiHub/Table/TableClientValueProvider.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Reflection;
+using Microsoft.Azure.WebJobs.Host.Bindings;
+
+namespace Microsoft.Azure.WebJobs.Extensions.ApiHub.Table
+{
+    internal class TableClientValueProvider : IValueProvider
```

Comment:
I like the one class per file approach. I won't change unless you feel strongly about it.